src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -167,6 +167,9 @@ object AddDatasetMetadataToDeposit {
         case (Some(q), None,   Some(t)) => elem(s"dcterms:$q")(t)
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
+        case _                          =>
+          // TODO this case needs to be checked to not occur in the preconditions (see also line 32)
```

Comment:
I know, but there is not really a way to refer to comments in another way, right? If you got a better way to refer to line 32, I'll be happy to change it!